src/reducer/async/actions.ts

```diff
@@ -97,12 +100,16 @@ function appendStateMetadata(
     appendedMetadata.description = state.fields.description;
   }
 
-  for (let row of state.metadataRows) {
+  for (let row of state.attributes) {
     if (row.name !== null && row.value !== null) {
-      appendedMetadata[row.name] = row.value;
+      if(!appendedMetadata.attributes) appendedMetadata.attributes = []
+      appendedMetadata.attributes.push({name: row.name, value: row.value});
```

Comment:
Later we will build a more robust metadata input form. But for now if the name matches a valid TZIP-021 field, it should populate that field. Otherwise, it should populate `attributes`. Thanks!

Reply:
Understood, I'll change to work that way.

Reply:
Previous behavior is changed. Before if a user added additional info they could use the fields to add additional TZIP-021 fields (externalURI, date, etc for example) since they were added to the top level of the metadata object. Now everything will be pushed into `attributes` and the user has no way of adding additional top level fields. This is not correct. Ideally if the added field name matches a TZIP-021 field, it will be added to the top level. If it does not it should be added to `attributes`. If this is not possible then the behavior should remain unchanged until we can implement better TZIP-021 support.

Reply:
Previous behavior is changed. Before if a user added additional info they could use the fields to add additional TZIP-021 fields (externalURI, date, etc for example) since they were added to the top level of the metadata object. Now everything will be pushed into attributes and the user has no way of adding additional top level fields. This is not correct. Ideally if the added field name matches a TZIP-021 field, it will be added to the top level. If it does not it should be added to attributes. If this is not possible then the behavior should remain unchanged until we can implement better TZIP-021 support.

Reply:
Alright, now I understood. I thought that that fields the user filled where exclusive for the attributes.